submissions/sprint_b/test_helpers.py

```diff
@@ -0,0 +1,79 @@
+from sprint_b.helpers import FileManager, Vocabulary, EpithetGenerator
+from flask import json
+import os
+import json
+import shutil
+
+"""     use pytest      """
+
+path = "../../resources/data.json"
+sad_path = ".fake./.path./resources123/data321.json"
+
+with open(os.path.abspath(path)) as json_data:
+    data = json.load(json_data)
```

Comment:
Test data such as this is called a fixture. Since we're using pytest, use pytest to define test fixtures instead: https://docs.pytest.org/en/latest/fixture.html#fixtures-as-function-arguments